consensus/XDPoS/engines/engine_v2/engine.go

```diff
@@ -942,24 +952,37 @@ func (x *XDPoS_v2) commitBlocks(blockCahinReader consensus.ChainReader, proposed
 	var decodedExtraField utils.ExtraFields_v2
 	err := utils.DecodeBytesExtraFields(parentBlock.Extra, &decodedExtraField)
 	if err != nil {
+		log.Error("Fail to execute first DecodeBytesExtraFields for commiting block", "ProposedBlockHash", proposedBlockHeader.Hash())
 		return false, err
 	}
 	if *proposedBlockRound-1 != decodedExtraField.Round {
+		log.Debug("[commitBlocks - 1] Rounds not continous found when committing block", "proposedBlockRound", proposedBlockRound, "decodedExtraField.Round", decodedExtraField.Round, "proposedBlockHeaderHash", proposedBlockHeader.Hash())
 		return false, nil
 	}
 
 	// If parent round is continous, we check grandparent
 	grandParentBlock := blockCahinReader.GetHeaderByHash(parentBlock.ParentHash)
 	err = utils.DecodeBytesExtraFields(grandParentBlock.Extra, &decodedExtraField)
 	if err != nil {
+		log.Error("Fail to execute second DecodeBytesExtraFields for commiting block", "parentBlockHash", parentBlock.Hash())
 		return false, err
 	}
 	if *proposedBlockRound-2 != decodedExtraField.Round {
+		log.Debug("[commitBlocks - 2] Rounds not continous found when committing block", "proposedBlockRound", proposedBlockRound, "decodedExtraField.Round", decodedExtraField.Round, "proposedBlockHeaderHash", proposedBlockHeader.Hash())
```

Comment:
could you just put `[commitBlocks]` instead of `[commitBlocks - 2]`, log with number is like personal debug sytle